src/main/java/uk/sky/cirrus/SchemaLoader.java

```diff
@@ -13,21 +13,21 @@
     private final Session session;
     private final String keyspace;
     private final SchemaUpdates schemaUpdates;
-    private final Paths paths;
+    private final CqlPaths paths;
 
-    public SchemaLoader(Session session, String keyspace, SchemaUpdates schemaUpdates, Paths paths) {
+    SchemaLoader(Session session, String keyspace, SchemaUpdates schemaUpdates, CqlPaths paths) {
         this.session = session;
         this.keyspace = keyspace;
         this.schemaUpdates = schemaUpdates;
         this.paths = paths;
     }
 
-    public void load() {
+    void load() {
         session.execute("USE " + keyspace + ";");
         paths.applyInSortedOrder(new Loader());
     }
 
-    private class Loader implements Paths.Function {
+    private class Loader implements CqlPaths.Function {
```

Comment:
We could implement the same without our own interface by using the Java BiFunction but we would need to return Void from the BiFunction::apply. To avoid that we have our custom Function interface which takes in the 2 arguments and returns void.